brain/morphology.cpp

```diff
@@ -61,7 +78,7 @@ class Morphology::Impl
         for( size_t i = 0; i != types->size(); ++i )
```

Comment:
None is possible (C++11 even less, because we can't use it in Brion).
Note that line 85 needs the index.